File: app/src/main/java/dev/ragnarok/fenrir/api/model/response/RepostReponse.java
package dev.ragnarok.fenrir.api.model.response;

import androidx.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

public class RepostReponse {

    @Nullable
    @SerializedName("post_id")
    public Integer postId;

    @Nullable
    @SerializedName("reposts_count")
    public Integer repostsCount;

    @Nullable
    @SerializedName("likes_count")
    public Integer likesCount;

}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/response/RepostReponse.kt
package dev.ragnarok.fenrir.api.model.response

import com.google.gson.annotations.SerializedName

class RepostReponse {
    @SerializedName("post_id")
    var postId: Int? = null

    @SerializedName("reposts_count")
    var repostsCount: Int? = null

    @SerializedName("likes_count")
    var likesCount: Int? = null
}